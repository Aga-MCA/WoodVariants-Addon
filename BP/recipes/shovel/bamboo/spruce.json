{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "bamboo_spruce_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:spruce_stick"
      },
      "P": {
        "item": "aga_wood_variants:bamboo_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:bamboo_spruce_shovel"
    }
  }
}