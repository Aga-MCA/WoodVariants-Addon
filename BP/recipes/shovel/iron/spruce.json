{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "iron_spruce_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:spruce_stick"
      },
      "P": {
        "item": "minecraft:iron_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:iron_spruce_shovel"
    }
  }
}