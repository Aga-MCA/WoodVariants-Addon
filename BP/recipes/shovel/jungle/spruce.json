{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "jungle_spruce_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:spruce_stick"
      },
      "P": {
        "item": "aga_wood_variants:jungle_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:jungle_spruce_shovel"
    }
  }
}