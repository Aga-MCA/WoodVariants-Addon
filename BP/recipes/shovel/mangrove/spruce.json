{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "mangrove_spruce_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:spruce_stick"
      },
      "P": {
        "item": "aga_wood_variants:mangrove_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:mangrove_spruce_shovel"
    }
  }
}