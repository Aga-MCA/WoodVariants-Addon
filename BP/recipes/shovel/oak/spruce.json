{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "oak_spruce_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:spruce_stick"
      },
      "P": {
        "item": "aga_wood_variants:oak_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:oak_spruce_shovel"
    }
  }
}