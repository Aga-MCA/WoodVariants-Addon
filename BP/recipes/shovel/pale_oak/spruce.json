{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "pale_oak_spruce_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:spruce_stick"
      },
      "P": {
        "item": "aga_wood_variants:pale_oak_panels"
      }
    },
    "result": {
      "item": "aga_wood_variants:pale_oak_spruce_shovel"
    }
  }
}