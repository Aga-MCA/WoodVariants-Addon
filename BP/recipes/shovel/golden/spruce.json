{
  "format_version": "1.12",
  "minecraft:recipe_shaped": {
    "description": {
      "identifier": "golden_spruce_shovel"
    },
    "tags": [
      "crafting_table",
      "aga"
    ],
    "pattern": [
      "P",
      "S",
      "S"
    ],
    "key": {
      "S": {
        "item": "aga_wood_variants:spruce_stick"
      },
      "P": {
        "item": "minecraft:gold_ingot"
      }
    },
    "result": {
      "item": "aga_wood_variants:golden_spruce_shovel"
    }
  }
}